sg_ssh/main.tf

```diff
@@ -0,0 +1,30 @@
+// Security Group Resource for Module
+resource "aws_security_group" "main_security_group" {
+  name        = "${var.security_group_name}"
+  description = "Security Group ${var.security_group_name}"
+  vpc_id      = "${var.vpc_id}"
+
+  // allows traffic from the SG itself for tcp
+  ingress {
```

Comment:
@antonbabenko Basically, you are right. It should be *only* `tcp/22`.
If not; it will be the same as having `any:any`, which is obviously, bad.